contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -278,6 +278,16 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
         optionsPurchaseQueue = newOptionsPurchaseQueue;
     }
 
+    /**
+     * @notice Sets oToken Premium
+     * @param minPrice is the new oToken Premium
+     */
+    function setMinPrice(uint256 minPrice) external onlyOwner {
+        require(minPrice > 0, "!minPrice");
+        ShareMath.assertUint104(minPrice);
+        currentOtokenPremium = uint104(minPrice);
```

Comment:
We don't actually need this assertion because the the `currentOtokenPremium` is stored as a uint256. So we can just do:

```solidity
currentOtokenPremium = minPrice
```